test/integration/runner_common.cpp

```diff
@@ -119,6 +121,48 @@ int IntegrationBase::exit_code(Proc &proc) {
   return proc.proc.exit_code();
 }
 
+fs::path IntegrationBase::createCustomConfig(std::string extraConfig) {
```

Comment:
```suggestion
fs::path IntegrationBase::createCustomConfig(std::string_view extraConfig) {
```
(or `const std::string &`)
If we don't need a copy of the string (depends on the boost trim thing)